src/Input/Select/Select.tsx

```diff
@@ -333,6 +334,7 @@ const Select: ISelect = (props: Props) => {
           small={small}
           disableTyping={disableTyping}
           readOnly={disableTyping}
+          autoFocus={defaultOpen}
```

Comment:
what will happen if we have 2 Select components on the same page, and they are both `defaultOpen = true`?

Reply:
Yes, so usually we won't just pass `ture` to `defaultOpen` when there are multip Selects.
e.g. `<Select defaultOpen={isLastSelect} />`